lib/zone.ts

```diff
@@ -1338,7 +1343,19 @@ const Zone: ZoneType = (function(global: any) {
     return '__zone_symbol__' + name;
   }
 
-
   performanceMeasure('Zone', 'Zone');
+  const z: any = Zone.prototype;
+  z.w = z.wrap;
```

Comment:
I would prefer that we don't pollute the Zone public API with these short names. I realized they are not part of `.d.ts` but they will show up in debugger.

How about this idea
`Zone.current` could be replaced with `getZoneCurrent()` which is declared in `util` file. The uglify will rename the `getZoneCurrent` to something short like `x`.

`Zone.current.wrap(...)` could be `wrapWithCurrentZone(...)`

and so on. What do you think?

Reply:
Yes, this is a better solution, I will create those helper functions.